### ISSUE TITLE
[Bug] sw_weight is parsed but the DCL term is commented out in the main training entry

### ISSUE THREAD

**Issue by gss10282025** [Jiahao004/SimCSE-plus-plus#3]
### Summary

`train_re_vicreg.py`, the entry the README points to via `run_re_vicreg.sh`, reads `sw_weight`, but in `simcse/models_re_vicreg.py` both the DCL computation and its addition to the loss are commented out:

```python
weight = cls.model_args.sw_weight
...
def dcl_loss(z1, z2):
    z1, z2 = cls.bn(z1), cls.bn(z2)
    ...
# cov_loss = dcl_loss(z1, z2)
...
# loss += weight * cov_loss
```

So `sw_weight=0.1` (the default) trains exactly the same model as `sw_weight=0`. Training runs without error, which makes the missing term easy to miss.

### Environment

Commit [51d2db55](https://github.com/Jiahao004/SimCSE-plus-plus/blob/51d2db55cf6009445558c9c9912ec887b6eb94a4/simcse/models_re_vicreg.py), PyTorch 2.8.0+cu128, RTX 5090, FP64.

### Steps to reproduce

Run the main entry for one update with `sw_weight=0.1`, `sw_temp=5`, `sampling_strategy="None"`, `do_mlm=False`, then again with `sw_weight=0`. One-layer BERT (hidden size 32, vocabulary 128), four two-view examples of length 8, dropout off, AdamW lr `0.001`, eps `1e-8`, no weight decay or clipping. Model and data loading were pointed at a local random checkpoint and fixed examples.

The two runs are identical. Both differ from a reference that includes the DCL term:

| Current code vs. reference with DCL | Normalized L2 difference | After restoring DCL |
| --- | ---: | ---: |
| Full gradient | `0.4764284153` | `6.94137e-14` |
| Parameter update (BatchNorm buffers excluded) | `0.3428845886` | `1.53129e-13` |

The BatchNorm buffers and counters are checked separately. Reproduced on a second host.

### Expected behavior

A positive `sw_weight` should add the weighted DCL term to the loss.

### Proposed fix

Restore the two commented lines, guarded on the weight:

```python
if weight > 0:
    cov_loss = dcl_loss(z1, z2)
    loss += weight * cov_loss
```

The computation and the addition sit in two different places in the file; both need restoring. Guarding the computation matters too: `dcl_loss` runs `cls.bn` in training mode, so calling it at zero weight would still update the BatchNorm running statistics.

This is separate from the collator issue in `train.py`. I did not evaluate STS or try to reproduce the paper's tables.
